parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -614,6 +616,17 @@ should_skip_binary_cleanup() {
     echo "${should_skip,,}"
 }
 
+should_enforce_kube_pmc_install() {
+    set -x
+    body=$(curl -fsSL -H "Metadata: true" --noproxy "*" "http://169.254.169.254/metadata/instance?api-version=2021-02-01")
```

Comment:
nit: make the TAG string a const so we can easily find it at the top Vs having it hidden as part of a big command line. 

`ShouldEnforceKubePMCInstall`